airbyte-ci/connectors/connector_ops/pyproject.toml

```diff
@@ -17,7 +17,7 @@ GitPython = "^3.1.29"
 pydantic = "^1.9"
```

Comment:
What holds our back to switch to the pydantic 2.0.0? Just curious?

Reply:
I don't know, nothing I'm aware of, but it's a different package / project than the current one :)

Reply:
I don't yet know the full surface area of us using Pydantic. If we're using 1.* everywhere, should we treat 2.0+ upgrade as a separate lane of work, or do you mostly expect things to work smoothly with such an upgrade?

Is is drop-in compatible syntax-wise?

If we're using it in the CDK itself and in airbyte-ci — I think airbyte-ci could be our guinea pig for such a migration? /cc @bazarnov